testapp/hybrid/app/main.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
```

Comment:
These files probably shouldn't be checked in, and should be generated instead. They won't change all that often so I don't care strongly about it for now though.